app/src/main/java/de/westnordost/streetcomplete/map/MapFragment.kt

```diff
@@ -116,7 +116,7 @@ open class MapFragment : Fragment(),
         mapTileProviderLink.text = vectorTileProvider.copyrightText
         mapTileProviderLink.setOnClickListener { showOpenUrlDialog(vectorTileProvider.copyrightLink) }
 
-        attributionContainer.respectSystemInsets(View::setMargins)
+        attributionContainer.respectSystemInsets { setMargins(it.left, it.top, it.right, it.bottom) }
```

Comment:
For here and other places, I think it would be more convenient if there was an extension function that accepts an `Insets`. I.e. something like

```kotlin
fun View.setMargins(insets: Insets) = 
  setMargins(insets.left, insets.top, insets.right, insets.bottom)
```
and the same for `setPadding`

Reply:
(There is a View.kt somewhere in utils where you could add that function)

Reply:
Sure.